src/bp/ui-shared/src/telemetry/offlineServerFallback.ts

```diff
@@ -0,0 +1,49 @@
+import axios, { AxiosInstance } from 'axios'
+import { TelemetryEvent } from 'common/telemetry'
+import ms from 'ms'
+
+export const sendEventsToTelemetryServer = async (events: TelemetryEvent[]) => {
```

Comment:
Use the short names

Reply:
@allardy that comes from me, do you have any better suggestioms ?

Reply:
Already fixed in 7e3743e!